index.yaml

```diff
@@ -38,7 +38,17 @@ entries:
   registry:
   - apiVersion: v2
     appVersion: 2.8.1
-    created: "2023-02-20T09:52:26.989615+01:00"
+    created: "2023-02-21T10:06:14.786141+01:00"
+    description: A Helm chart for OSISM offline mirrors -  registry (containers)
+    digest: 283b4cedf362942798f5347b5e9fd63649d5eb329ce7cc9a049624e8b2450410
+    name: registry
+    type: application
+    urls:
+    - https://osism.github.io/helm-charts/registry-0.1.1.tgz
+    version: 0.1.1
+  - apiVersion: v2
+    appVersion: 2.8.1
+    created: "2023-02-21T10:06:14.785805+01:00"
```

Comment:
There are duplicate keys in here. Is this correct? Doesn't look right to me.

Reply:
The keys apiVersion (line 39 + 49), appVersion (40 + 50), created (41 + 51 with two different values), description (42 + 52), digest (43 + 53 with two different values) and name (44 + 54) are duplicate but it probably didn't trigger in the linter because of the prepending dash at Line 49. To me this doesn't look right.

Reply:
ah, now I get it. The only difference is found in the "version" key: 0.1.0 -> 0.1.1. The rest is totally fine, even tough it looks kind of wrong. The "version" key is the major indicator for helm to indicate, if an upgrade of the chart is needed or not.

Reply:
Are you sure that it's correct? Because of the commit gets merged the resulting YAML would look like this:
```
registry:
  - apiVersion: v2
    appVersion: 2.8.1
    created: "2023-02-21T10:06:14.786141+01:00"
    description: A Helm chart for OSISM offline mirrors -  registry (containers)
    digest: 283b4cedf362942798f5347b5e9fd63649d5eb329ce7cc9a049624e8b2450410
    name: registry
    type: application
    urls:
    - https://osism.github.io/helm-charts/registry-0.1.1.tgz
    version: 0.1.1
  - apiVersion: v2
    appVersion: 2.8.1
    created: "2023-02-21T10:06:14.785805+01:00"
    description: A Helm chart for OSISM offline mirrors -  registry (containers)
    digest: fee7e555aeec0f4c858d1ab27f53d63be469f87967d3f4b176b44728442b8d7c
    name: registry
```